yelp_package/extra_requirements_yelp.txt

```diff
@@ -15,7 +15,6 @@ pyjwt==2.9.0               # required by okta-auth
 pyopenssl==23.2.0          # vault-tools dependency
 saml-helper==2.5.3         # required by okta-auth
 service-identity==24.2.0   # vault-tools dependency
-simplejson==3.19.2         # required by tron CLI
```

Comment:
we don't support python2, we don't need simplejson anymore and can just rely on the stdlib